private_repo/ansible/inventory/dev/Core/common.yml

```diff
@@ -20,41 +20,102 @@ sunbird_mail_server_from_email: "support@myorg.com"       # Email ID that should
+cloud_storage_dial_bucketname: ""
+# This storage contains flink checkpoint data
+# Example: cloud_storage_flink_bucketname: "flink-storage"
+cloud_storage_flink_bucketname: ""
+# This storage contains portal cdn file
+# Example: cloud_storage_flink_bucketname: "playercdn-storage"
+cloud_storage_playercdn_bucketname: ""
+# This storage contains public data
+# Example: cloud_storage_flink_bucketname: "public-storage"
+cloud_storage_public_bucketname: ""
+# This storage contains public reports data
+# Example: cloud_storage_flink_bucketname: "publicreports-storage"
+cloud_storage_publicreports_bucketname: ""
+# This storage contains private reports data
+# Example: cloud_storage_flink_bucketname: "privatereports-storage"
+cloud_storage_privatereports_bucketname: ""
+# This storage contains samiksha data
+# Example: cloud_storage_flink_bucketname: "samiksha-storage"
+cloud_storage_samiksha_bucketname: ""
+# This storage contains schema data
+# Example: cloud_storage_flink_bucketname: "schema-storage"
+cloud_storage_schema_bucketname: ""
+# This storage contains sourcing related data
+# Example: cloud_storage_flink_bucketname: "sourcing-storage"
+cloud_storage_sourcing_bucketname: ""
+# This storage contains desktop app data
+# Example: cloud_storage_flink_bucketname: "offlineinstaller-storage"
+cloud_storage_offlineinstaller_bucketname: ""
+# This storage contains public schemas, contents
+# Example: cloud_storage_flink_bucketname: "content-storage"
+cloud_storage_content_bucketname: ""
+# This storage contains telemetry data
+# Example: cloud_storage_flink_bucketname: "telemetry-storage"
+cloud_storage_telemetry_bucketname: ""
+# This storage contains T & C data
+# Example: cloud_storage_flink_bucketname: "termsandconditions-storage"
+cloud_storage_termsandcondtions_bucketname: ""
+# Example: cloud_storage_flink_bucketname: "user-storage"
+cloud_storage_user_bucketname: ""
+# This storage contains crashlogs
+# Example: cloud_storage_flink_bucketname: "desktopappcrashlogs-storage"
+cloud_storage_desktopappcrashlogs_bucketname: ""
+# This storage contains labels data
+# Example: cloud_storage_flink_bucketname: "label-storage"
+cloud_storage_label_bucketname: ""
+# Example: cloud_storage_flink_bucketname: "certservice-storage"
+cloud_storage_certservice_bucketname: ""
+# This storage contains UCI services data
+# Example: cloud_storage_flink_bucketname: "uci-storage"
+cloud_storage_uci_bucketname: "" 
+# This storage contains artifacts data
+# Example: cloud_storage_flink_bucketname: "artifact-storage"
+cloud_storage_artifacts_bucketname: ""
+# This storage contains backups data
+# Example: cloud_storage_flink_bucketname: "management-storage"
+cloud_storage_management_bucketname: ""
 
 # Uncomment the variable based on your cloud provider (as a default we have kept Azure variable uncommented)
 # GCP
-# cloud_storage_url: https://storage.cloud.google.com/{{ gcloud_public_bucket_name }}
+# cloud_storage_url: https://storage.googleapis.com
```

Comment:
How is this getting injected into services? The URL is missing bucket info.
Does `cloud_storage_url` point to the generic CSP url and we append in microservices configs as `{{ cloud_storage_url }} / {{ bucket }}` ? Do verify this. I am not 100% we are injecting bucket value separately in configs.